participants/artur_ptaszek.json

```diff
@@ -0,0 +1,20 @@
+{
+	"name": "Artur Ptaszek",
+	"company": "Port Blue Sky",
+	"when": {
+		"friday": true,
+		"saturday": true
+	},
+	"iCanTakeNotesDuringSessions": false,
+	"tags": ["React", "TypeScript", "TanStack Query", "Rust", "tmux", "VIM", "k8s", "Docker", "OpenID Connect"],
+	"vegan": false,
+	"vegetarian": false,
+	"whatIsMyConnectionToJavascript": "I work TS/JS Full Stack for over 10 years, but as a enthusiast much more.",
+	"whatCanIContribute": "Generating a client code based on OpenAPI spec, Doken - my tool for getting OAuth2/OpenID connect token directly in a console for all authentication grants, contributing in discussions.",
+	"tShirt": {
+		"size": "L",
+		"type": "regular"
+	},
+	"twitter": "RiddleMan",
+	"website": "https://ptaszek.dev"
```

Comment:
Oh noes! Looks like the website is not available - you still want to keep it?
![image](https://user-images.githubusercontent.com/1767865/236677085-1c58662c-1b90-442b-9c60-1c173db2a78b.png)

Reply:
haha I did that with the intention of finally adding anything in there :D

Reply:
Very well! We'll merge it. I'm going to add the links to the social accounts and website later today, so you should still have a little bit of time before you get links to a dead site 😉